### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by zupo**

### CODE REVIEW

File: docs/schedule.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 ## Sunday
 
-Welcome party at 7PM outside at [La Tabla](https://maps.app.goo.gl/FPRX4NZy4LkGmcFX6).
+Welcome party at 7PM at [La Tabla](https://maps.app.goo.gl/FPRX4NZy4LkGmcFX6).
 
 ## Monday
 
-- 8AM: venue opens
+- 8AM: [venue](https://house.niteo.co/) opens
```
Comment (zupo) [inline docs/schedule.md]:
```suggestion
- 8AM: [Sprint Venue](https://house.niteo.co/) opens
```

File: docs/schedule.md [hunk 2]
```diff
@@ -21,16 +21,16 @@ Welcome party at 7PM outside at [La Tabla](https://maps.app.goo.gl/FPRX4NZy4LkGm
 - 1PM: lunch
 - 3PM: activity
 - 7PM: standup
-- 7PM: [Luis Guachinche](https://maps.app.goo.gl/aWmjG5qkFbmBYRKr6) catering
+- 7PM: [Luis Guachinche](https://maps.app.goo.gl/aWmjG5qkFbmBYRKr6) paella @ vanue
```
Comment (domenkozar) [inline docs/schedule.md]:
```suggestion
- 7PM: [Luis Guachinche](https://maps.app.goo.gl/aWmjG5qkFbmBYRKr6) paella @ venue
```

File: docs/schedule.md [hunk 3]
```diff
@@ -48,5 +48,5 @@ Welcome party at 7PM outside at [La Tabla](https://maps.app.goo.gl/FPRX4NZy4LkGm
 - 9AM: breakfast
 - 1PM: lunch
 - 2PM: sprint wrap up
-- 3PM: venue closes, wine tasting at [Bodega Stratvs](https://stratvs.com/)
-- 7PM: farewell dinner outside at [La Tavernetta](https://maps.app.goo.gl/4cvTr3c2DYHaGvUT6)
+- 3PM: venue closes, leave for wine tasting at [Bodega Stratvs](https://stratvs.com/)
```
Comment (domenkozar) [inline docs/schedule.md:51]:
```suggestion
- 3PM: venue closes, leave for wine tasting to [Bodega Stratvs](https://stratvs.com/)
```

Comment (zupo) [reply docs/schedule.md:51]: I prefer "at" but I don't have a strong opinion.

**Pull request merged**